vector/src/main/res/values/strings.xml

```diff
@@ -2528,6 +2528,16 @@
     <string name="ftue_auth_carousel_4_title">Cut the slack from teams.</string>
     <string name="ftue_auth_carousel_4_body">As universal as email, Element is a completely new type of collaboration.</string>
 
+    <string name="ftue_auth_use_case_title">Who will you chat to the most?</string>
+    <string name="ftue_auth_use_case_subtitle">We\'ll help you get connected.</string>
+    <string name="ftue_auth_use_case_option_one">Friends and family</string>
+    <string name="ftue_auth_use_case_option_two">Teams</string>
+    <string name="ftue_auth_use_case_option_three">Communities</string>
+    <string name="ftue_auth_use_case_skip">Not sure yet? %s</string>
+    <string name="ftue_auth_use_case_skip_partial">You can skip this question</string>
+    <string name="ftue_auth_use_case_join_existing_server">Looking to join an existing server?</string>
+    <string name="ftue_auth_use_case_connect_to_server">Connect to server</string>
```

Comment:
It's OK, but you were thinking about adding new strings to another file waiting for final copy.

Reply:
:100: will do

Reply:
83af0d7bb9c0cd7bf96750d3cd4778aa470c8357